lib/include/hse_ikvdb/csched.h

```diff
@@ -59,7 +59,9 @@ enum cn_rule {
     CN_RULE_SCATTERF,       /* vgroup scatter remediation (full node) */
     CN_RULE_SCATTERP,       /* vgroup scatter remediation (partial node) */
     CN_RULE_GARBAGE,        /* leaf garbage (reducing space amp) */
-    CN_RULE_SPLIT,          /* big leaf (near split threshold) */
+    CN_RULE_SPLIT,          /* big leaf (near split threshold, split in progress) */
+    CN_RULE_LSPLIT,         /* left node kvset after a split */
+    CN_RULE_RSPLIT,         /* right ndoe kvset after a split */
```

Comment:
I don't see any code in this PR that uses this rule.

Reply:
It's done via math in some function, which relies on these enumerators being in order.  I can fix that to make it not do that, then you'd see where they are used...

Reply:
Yeah... asking for trouble if you depend on their order in the enum.

Reply:
Fixed in cn_comp_commit().